### ISSUE TITLE
Silent Errors

### ISSUE THREAD

**Issue by clearly** [mulesoft/api-console#178]
I'm using the latest code checked into dist

I got the console working with a simple api.raml file.

When I put in a complex/sophisticated api.raml file the file never renders (bouncy bars the whole time). The complex api.raml file passes ramllint. No errors are printed in the console, stdout, or stderr.

Any idea how to move forward?

**Comment by dmartinezg** [comment]
Hi @clearly, can you share the RAML which fails for you? if you can't can you provide us with a smaller one which still produces the outcome?

**Comment by clearly** [comment]
Can I send privately? It is internal corporate proprietary stuff.

**Comment by dmartinezg** [comment]
Yes, my email is public on my github profile

**Comment by jcenturion** [comment]
@clearly: I've already have a [PR](https://github.com/mulesoft/api-console/pull/179) for fixing this. I'll let you know once it is merged into `master`.

**Comment by dmartinezg** [comment]
Great #179 should fix this, thanks

**Comment by jcenturion** [comment]
@clearly: It's already on `master` ;).
